integreat_cms/core/settings.py

```diff
@@ -114,6 +114,11 @@
     os.environ.get("INTEGREAT_CMS_DEFAULT_REQUEST_TIMEOUT", 10)
 )
 
+#: The URLs which are treated as internal in TinyMCE custom link plugin
+CUSTOM_INTERNAL_URLS = os.environ.get("INTEGREAT_INTERNAL_URLS", "").split()
+INTERNAL_URLS = " ".join(
+    url for url in ["integreat.app", ".integreat-app.de"] + CUSTOM_INTERNAL_URLS if url
+)
```

Comment:
```suggestion
#: The URLs which are treated as internal in TinyMCE custom link plugin
INTERNAL_URLS = [BASE_URL, WEBAPP_URL, WEBSITE_URL] + list(
    filter(
        None,
        (
            x.strip()
            for x in os.environ.get("INTEGREAT_CMS_INTERNAL_URLS", "").splitlines()
        ),
    )
)
```
1. I wouldn't hardcode "integreat.app" and "integreat-app.de" here since we use the same code for whitelabeled versions like the malte app and the aschaffenburg app. Instead, we can rely on the existing settings for these domains. Also, this requires to move either this setting down or the `BASE_URL` up so it can be used here.
2. To stay consistent with our other settings which contain lists (e.g. [`ALLOWED_HOSTS`](https://github.com/digitalfabrik/integreat-cms/blob/c87c51f80943dfc58cc249f698d85dfae1a1ad5b/integreat_cms/core/settings.py#L336-L346) or [`LANGUAGES`](https://github.com/digitalfabrik/integreat-cms/blob/c87c51f80943dfc58cc249f698d85dfae1a1ad5b/integreat_cms/core/settings.py#L649-L662)), I would split by newline characters instead of spaces, even if this makes the code a bit less readable. This makes it possible to write e.g.
    ```
    INTERNAL_URLS =
	    internal.host.org
	    my.app.de
    ```
into the config file. Speaking of which, could you add an example of this to [example-configs/integreat-cms.ini](https://github.com/digitalfabrik/integreat-cms/blob/develop/example-configs/integreat-cms.ini)?

Reply:
Good point, this looks way more readable! :+1:
Could you change the syntax of the other list settings as well? :wink:

Reply:
Please have a look if you're happy with this. Unfortunately `BASE_URL` and so forth all come with an attached `https://`, but we need to not only capture `https://integreat-app.de` but also (for example) `cms.integreat-app.de` as @svenseeberg has already remarked, so I had to do some slight post-processing.

Reply:
Ah okay, I thought the `BASE_URL` would already account for the CMS url. But yes, marking all subdomains as internal as well makes sense I guess.